### ISSUE TITLE
Allow navigating all hunks in `blame_line()`

### ISSUE THREAD

**Issue by lewis6991** [lewis6991/gitsigns.nvim#366]

**Comment by ecosse3** [comment]
Does it mean that I would be able to go to the previous/next commit on a specific line I have `blame_line{full=true}` opened? I am very missing this feature  (was using it in git-messenger).

**Comment by lewis6991** [comment]
Nope. It just means you can view all hunks of the blame commit. See #198 for cycling blame history.

**Comment by Mephistophiles** [comment]
It will be helpful for me. It would also be nice to enter a pop-up window created by the command.
